commit cc1ae1f9207c81c89a9daef070e195338c5b9d62
Author: Maxime Ripard <maxime.ripard@free-electrons.com>
Date:   Mon Jan 23 11:41:46 2017 +0100

    rtc: sun6i: Disable the build as a module
    
    commit 3753941475ae6501dcd1e41832bd0e6c35247d6a upstream.
    
    Since we have to provide the clock very early on, the RTC driver cannot be
    built as a module. Make sure that won't happen.
    
    Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1342_cc1ae1f9207c81c89a9daef070e195338c5b9d62.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1342_cc1ae1f9207c81c89a9daef070e195338c5b9d62.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
